### PR TITLE
Fix setup / remove plone 4.2 and add plone 5.1 + plone.api min. requirement. 

### PR DESCRIPTION
**Pull request by maethu**

### CODE REVIEW

**Comment by maethu** [comment]
In order to let the plone 5.1.x pass https://github.com/collective/collective.geo.mapwidget/pull/31 is necessary.

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,7 +5,11 @@ Changelog
 1.3.7 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Drop Plone 4.2 support. [mathias.leimgruber]
+
+- Since Version 1.3.6 plone.api 1.5.1 us required. [mathias.leimgruber]
```
Comment (jone) [inline docs/HISTORY.txt]: `us` -> `is`

File: ftw/geo/tests/test_archetypes.py [hunk 2]
```diff
@@ -29,6 +31,7 @@ def getLocationString(self):
         return self.context.Title()
 
 
+@skipIf(IS_PLONE_5, 'Disale AT tests for Plone 5.1')
```
Comment (jone) [inline ftw/geo/tests/test_archetypes.py]: `Disale` -> `Disable`

File: docs/HISTORY.txt [hunk 3]
```diff
@@ -5,7 +5,11 @@ Changelog
...
+
+- Add Plone 5.1 text config. [mathias.leimgruber]
```
Comment (jone) [inline docs/HISTORY.txt]: What do you mean by `text config`?

**Pull request merged**